src/content-tags-drawer/data/api.js

```diff
@@ -28,7 +28,7 @@ export const getTaxonomyTagsApiUrl = (taxonomyId, options = {}) => {
 
   return url.href;
 };
-export const getContentTaxonomyTagsApiUrl = (contentId) => new URL(`api/content_tagging/v1/object_tags/${contentId}/`, getApiBaseUrl()).href;
+export const getContentTaxonomyTagsApiUrl = (contentId) => new URL(`api/content_tagging/v1/object_tags/${contentId}/?include_perms`, getApiBaseUrl()).href;
```

Comment:
We don't need the `include_perms` because we will use the Taxonomy `canTagObject` and not the Tag permission. The Tag permission will be used to add/remove Tags to a Taxonomy.

Reply:
We need the `canDeleteObjecttag` permission that comes from this object_tags endpoint, so we need its response to include permissions.

But you're right -- we could also get `canTagObject` permission from the taxonomy endpoint, either works.